packages/cli-repl/src/update-notification-manager.ts

```diff
@@ -1,10 +1,10 @@
 import semver from 'semver';
 import { promises as fs } from 'fs';
-import type { RequestInfo, RequestInit, Response } from 'node-fetch';
+import importNodeFetch from '@mongosh/import-node-fetch';
 
 // 'http' is not supported in startup snapshots yet.
-const fetch = async (url: RequestInfo, init?: RequestInit): Promise<Response> =>
-  await (await import('node-fetch')).default(url, init);
+const fetch = async (url: any, init?: any): Promise<any> =>
+  await (await importNodeFetch()).default(url, init);
```

Comment:
I don't think we need this level of indirection anymore, we should be able to just call `importNodeFetch()` directly where we call `fetch()` below.

If this is necessary, well, let's maybe re-export the necessary types from `@mongosh/import-node-fetch`, it wouldn't be great to have to stick with `any` here

Reply:
Yeah, so, since we re-enabled startup snapshot support one of the things we cannot do anymore is to have imports of `http`/`https`/`tls` and a few other built-in Node.js modules at the top level (i.e. loaded when mongosh initially starts). That's hopefully a restriction that's going to go away in #1846, but, well, one step at a time. :)